src/view/main.js

```diff
@@ -60,6 +61,9 @@ class MainView extends Component {
         {route === 'SeedVerify' && (
           <SeedVerify store={store} nav={nav} wallet={wallet} />
         )}
+        {route === 'RestoreWallet' && (
+          <RestoreWallet store={store} nav={nav} wallet={wallet} />
+        )}
```

Comment:
I'd suggest we wait with inserting the new screen into main.js until we have the logic wired up.

Reply:
fixed :)